src/proxymatic/test/marathon/testLoadBalancerOptions/_v2_apps_demo_webapp_versions_2016-10-04T12:48:17.435Z.json

```diff
@@ -31,7 +31,9 @@
   "labels": {
     "com.meltwater.proxymatic.port.0.weight": "100",
     "com.meltwater.proxymatic.port.0.maxconn": "150",
-    "com.meltwater.proxymatic.port.0.mode": "http"
+    "com.meltwater.proxymatic.port.0.mode": "http",
+    "com.meltwater.proxymatic.service.timeoutclient": 500,
```

Comment:
Should be 

```
    "com.meltwater.proxymatic.port.0.timeout.client": 250,
    "com.meltwater.proxymatic.port.0.timeout.server": 500,
```

Would suggest setting them to different values in the test to make sure they're not mixed up somewhere in the code